### PR TITLE
add CloudWatch

### PR DESCRIPTION
**Pull request by nick-otter**

### CODE REVIEW

File: playbook.yml [hunk 1]
```diff
@@ -113,3 +114,56 @@
...
+          datetime_format = %b %d %H:%M:%S
+
+    - name: Install cloudwatch log agent
+      environment:
+        LC_ALL: C
+      shell: /tmp/awslogs-agent-setup.py --region eu-west-2 --non-interactive -c /tmp/cwlogs.conf
+      args:
+        creates: /var/awslogs/etc/aws.conf
+
+    - name: Copy cloudwatch logs starter
```
Comment (chrisns) [inline playbook.yml:150]: you will need this file

File: playbook.yml [hunk 2]
```diff
@@ -113,3 +114,56 @@
...
+        group: root
+        mode: 0755
+
+    - name: Make Cron Job to start awslogs with right region
+      cron:
+        name: Start awslogs
+        special_time: reboot
+        job: /usr/bin/startcloudwatchlogs.sh
+
+    - name: Bootstrap
```
Comment (chrisns) [inline playbook.yml]: you don't have ephemeral instance stores on this machine?

**Review by chrisns — CHANGES_REQUESTED** [review]

File: playbook.yml (hunk 1 above)
Comment (nick-otter) [reply playbook.yml:150]: now added

File: startcloudwatchlogs.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# Get the instance region and inject it in the conf
+REGION=$(ec2metadata --availability-zone | head -c-2)
```
Comment (chrisns) [inline startcloudwatchlogs.sh:3]: is ec2metadata present on the machine, you may need to install it, think its cloud-utils maybe

**Review by chrisns — CHANGES_REQUESTED** [review]

File: startcloudwatchlogs.sh (hunk 3 above)
Comment (nick-otter) [reply startcloudwatchlogs.sh]: according to [this](https://medium.com/@jbirdvegas/query-ec2-metadata-on-ubuntu-dc6cfbe17940) it is cloud utils, will add

Comment (nick-otter) [reply startcloudwatchlogs.sh:3]: it wasn't, have now added

File: playbook.yml [hunk 4]
```diff
@@ -80,12 +82,12 @@
       args:
         chdir: /lib
       with_items:
-        - git clone https://github.com/UKHomeOffice/data-transfer
+        - test -e /lib/data-transfer || git clone https://github.com/UKHomeOffice/data-transfer
```
Comment (chrisns) [inline playbook.yml]: I'm not sure this will work? have you tested it?

Comment (chrisns) [reply playbook.yml]: http://docs.ansible.com/ansible/latest/git_module.html

**Review by chrisns — CHANGES_REQUESTED** [review]

File: playbook.yml (hunk 4 above)
Comment (nick-otter) [reply playbook.yml]: it does, but was for running the playbook multiple times so not necessarily needed - I will refactor to use the ansible git module too

**Review by chrisns — APPROVED** [review]

**Pull request merged**